File: ada/ada2022/Python3/TypeClassification.py
from enum import Enum, auto

class TypeClassification(Enum):
    Global_ = auto()
    Block_ = auto()
    TypeName_ = auto()
    ObjectName_ = auto()
    SubprogramName_ = auto()
    PackageName_ = auto()
    ExceptionName_ = auto()
    GenericName_ = auto()
    EnumerationLiteral_ = auto()
    ComponentName_ = auto()
    LabelName_ = auto()

File: ada/ada2022/Go/TypeClassification.go
package parser

type TypeClassification int

const (
	Global_ TypeClassification = iota
	Block_
	TypeName_
	ObjectName_
	SubprogramName_
	PackageName_
	ExceptionName_
	GenericName_
	EnumerationLiteral_
	ComponentName_
	LabelName_
)
